s3/s3_operations.cpp

```diff
@@ -1056,7 +1058,7 @@ namespace irods_s3 {
         S3_delete_object(
             &bucketContext,
             key.c_str(), 0,
-            0,                    //timeout
+            1800000,                    //timeout
```

Comment:
Should we capture this in a constant somewhere? Maybe even a config?

Reply:
It is now a configuration parameter with a default that is a constant.